modules/commons/src/test/java/org/apache/synapse/commons/json/JsonStreamingBuilderTest.java

```diff
@@ -19,8 +19,36 @@
 package org.apache.synapse.commons.json;
 
 import junit.framework.TestCase;
+import org.apache.axiom.om.OMElement;
+import org.apache.axis2.builder.Builder;
+import org.apache.axis2.context.MessageContext;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import java.io.InputStream;
 
```

Comment:
Shall we add little description about the test case even though this is an old empty test-case